crates/cheatnet/tests/contracts/Scarb.toml

```diff
@@ -5,7 +5,6 @@ version = "0.1.0"
 # See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest
 
 [[target.starknet-contract]]
-sierra = true
 casm = true
```

Comment:
sierra is enabled by default, you have to do `sierra = false` to truly check it. I think we should leave some tests with sierra enable too (at least one) to make sure we always parse the artifacts properly

Reply:
Some scarb unit tests still use `sierra = true`, I've added a comment explaining that to `scarb::setup_package`.